pkgs/development/python-modules/markdown2/default.nix

```diff
@@ -1,15 +1,23 @@
-{ lib, buildPythonPackage, fetchPypi }:
+{ lib, buildPythonPackage, fetchFromGitHub, python, pygments }:
 
 buildPythonPackage rec {
   pname = "markdown2";
-  version = "2.3.6";
+  version = "2.4.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    extension = "zip";
-    sha256 = "08a124043aa0ad36ba2136239547d5011a2b770278abb11a5609611e0040ea05";
+  # PyPI does not contain tests, so using GitHub instead.
+  src = fetchFromGitHub {
+    owner = "trentm";
+    repo = "python-markdown2";
+    rev = "${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
Thanks! Fixed.

(I did check my changes with `nixpkgs-review` myself but this suggestion didn't show up. I guess I was using an older version that didn't have this check yet)

Reply:
It is something extra I wrote https://github.com/SuperSandro2000/nixpkgs-review-checks